changelog.json

```diff
@@ -1,6 +1,18 @@
 {
   "repository": "newrelic/node-newrelic",
   "entries": [
+    {
+      "version": "11.17.0",
+      "changes": {
+        "security": [],
+        "bugfixes": [
+          "Added support for AWS API Gateway v2 payloads",
+          "Reverted openai instrumentation update to handle broken version of 4.40.1",
+          "Updated instrumentation to handle the different exports in 4.40.1 of `openai`"
+        ],
+        "features": []
```

Comment:
make sure you add the features i referenced above here too

Reply:
Updated